src/SFA.DAS.PR.Api.UnitTests/Controllers/AccountProvidersControllerGetTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using FluentValidation.Results;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.PR.Api.Common;
+using SFA.DAS.PR.Api.Controllers;
+using SFA.DAS.PR.Application.AccountProviders.Queries.GetAccountProviders;
+using SFA.DAS.PR.Application.Mediatr.Responses;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.PR.Api.UnitTests.Controllers;
+
+public class AccountProvidersControllerGetTests
+{
+    [Test]
+    [MoqAutoData]
+    public async Task GetProviders_InvokesQueryHandler(
+       [Frozen] Mock<IMediator> mediatorMock,
+       [Greedy] AccountProvidersController sut,
+       long accountId, 
+       CancellationToken cancellationToken
+    )
+    {
+        await sut.Get(accountId, cancellationToken);
+
+        mediatorMock.Verify(m => 
+            m.Send(It.Is<GetAccountProvidersQuery>(q => q.AccountId == accountId),
+            It.IsAny<CancellationToken>())
+        );
+    }
+
+    [Test]
+    [MoqAutoData]
```

Comment:
We need to be careful what response we are introducing as the existing consumers may not have accounted for it.